source/terminology.rst

```diff
@@ -2,6 +2,9 @@
 
 Terminology
 ===========
+.. Note:: 
+	Adam:
+	This section is great to have and very helpful. Some nomenclature from the theory section is missing. Some variables (such as A, B, C, d) are used differently in different parts of the documentation (PTO, BEM theory, etc), these needs subscripts and distinct mentions here for each different use of that variable
 
```

Comment:
I agree, we've tried to be consistent but please revise anything that is not appropriated documented.